stanovy.tex

```diff
@@ -108,16 +116,20 @@ \section{Členství}
     \item Práva aktivních členů:
     \begin{enumerate}[itemsep=0pt,topsep=0pt]
         \item užívat výhod aktivního člena Spolku, pokud jsou takové vyhlášeny,
-        \item volit a být volen do řídících a kontrolních orgánů Spolku.
+        \item volit a být volen do řídících orgánů Spolku.
+    \end{enumerate}
+    \item Práva emeritních členů:
+    \begin{enumerate}[itemsep=0pt,topsep=0pt]
+        \item užívat výhod aktivního člena Spolku, pokud jsou takové vyhlášeny.
```

Comment:
Tohle mi přijde smysluplná výhoda... Rovnou bych navrhl, že by do výhod mělo obecně patřit možnost půjčit si po dohodě věci ze skladu.